be/src/common/config.h

```diff
@@ -976,4 +976,10 @@ CONF_Int32(default_mv_resource_group_cpu_limit, "1");
 
 // Max size of key columns size of primary key table, default value is 128 bytes
 CONF_mInt32(primary_key_limit_size, "128");
+
+// If your sort key cardinality is very high,
+// You could enable this config to speed up the point lookup query,
+// otherwise, StarRocks will use zone map for one column filter
+CONF_mBool(enable_short_key_for_one_column_filter, "false");
```

Comment:
can we put it into session variable so that it can be tuned for each query?

Reply:
I considered this point, which is too hard for user. So I think we could set config for one cluster if this cluster is for point look up